src/content/advanced/cluster-management/node-pools-capi/index.md

```diff
@@ -0,0 +1,322 @@
+---
+linkTitle: Node pools (CAPI)
+title: Node pools (CAPI)
+description: A general description of node pools as a concept, its benefits, and some details you should be aware of.
+weight: 50
+menu:
+  main:
+    parent: advanced-cluster-management
+user_questions:
+  - What is a node pool?
+  - What are node pools?
+  - In which cloud environments are node pools supported?
+  - Which workload cluster releases introduced node pools?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+last_review_date: 2023-11-22
+---
+
+## Definition
+
+A node pool is a set of nodes within a Kubernetes cluster that share the same configuration (machine type, CIDR range, etc.). Each node in the pool is labeled by the node pool's name.
```

Comment:
` CIDR range` I would remove this as all nodes in whole cluster share same CIDR range,

Different CIDR range per nodepool was true for vintage